### PR TITLE
Fix Image::to_pixels()

### PR DESCRIPTION
**Pull request by Vixeliz**
Fixes: #1135

### CODE REVIEW

File: examples/cpu_image.rs [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+        canvas.draw(&self.image, Vec2::new(380.0, 380.0));
+
+        canvas.finish(ctx)?;
+
+        Ok(())
+    }
+}
+
+pub fn main() -> GameResult {
+    let cb = ggez::ContextBuilder::new("super_simple", "ggez");
```
Comment (sampullman) [inline examples/cpu_image.rs:46]: Loading the resource fails for me unless the resource path is added:

```rs
    let resource_dir = if let Ok(manifest_dir) = env::var("CARGO_MANIFEST_DIR") {
        let mut path = path::PathBuf::from(manifest_dir);
        path.push("resources");
        path
    } else {
        path::PathBuf::from("./resources")
    };

let cb = ggez::ContextBuilder::new("super_simple", "ggez").add_resource_path(resource_dir);
```

Comment (Vixeliz) [reply examples/cpu_image.rs:46]: Oh yeah thank you my bad

**Comment by sampullman** [comment]
Aside from my comment, I can confirm that this fixes the bug!

**Comment by Vixeliz** [comment]
> Aside from my comment, I can confirm that this fixes the bug!

Awesome after the checks complete then I'll merge it I fixed the example.

**Comment by Vixeliz** [comment]
I'm merging anyways as that failed due to alsa not cause of any rust issue. Plus it should run again when I merge

**Pull request merged**